sirinternal.c

```diff
@@ -1208,7 +1208,9 @@ bool _sir_getthreadname(char name[SIR_MAXPID]) {
     pthread_get_name_np(pthread_self(), name, SIR_MAXPID);
     return _sir_validstrnofail(name);
 #else
-# pragma message("unable to determine how to get a thread name")
+# if !defined(_AIX)
```

Comment:
I don't have AIX above, and there isn't any easy way I know of to get a thread name on AIX, so I don't want to warn about it.  I think it can be done, but it's pretty uncommon, AFAIK.

Reply:
oh, sorry, the folding confused me and I thought we were still in gettid